spinnaker/resource_pipeline_test.go

```diff
@@ -0,0 +1,105 @@
+package spinnaker
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/mercari/terraform-provider-spinnaker/spinnaker/api"
+)
+
+func TestAccResourceSourceSpinnakerPipeline_basic(t *testing.T) {
+	resourceName := acctest.RandomWithPrefix("tf-acc-test")
+	application := acctest.RandomWithPrefix("tf-acc-test")
+	pipeline := `{
+  		"keepWaitingPipelines": false,
+  		"limitConcurrent": true,
+  		"notifications": [ ],
+  		"stages": [ ],
+  		"triggers": [ ]
+		}`
```

Comment:
Can you remove the extra space, please 🙏 

```suggestion
	pipeline := `{
  		"keepWaitingPipelines": false,
  		"limitConcurrent": true,
  		"notifications": [],
  		"stages": [],
  		"triggers": []
		}`
```

Reply:
Hi, updated last commit. 
Awaiting Your approval for merge as I'm looking to make another pull request with terraform import feature for pipelines, and tests for this feature as well.

Reply:
Thank you for your update 👍